scripts/migration/db-restore.bash

```diff
@@ -38,7 +38,5 @@ if [ "$yn" = "y" ]; then
     -v "$(pwd)":/var/pgdata \
     jbergknoff/postgresql-client postgresql://"${POSTGRES_USER}":"${POSTGRES_PASSWORD}"@"${POSTGRES_HOST}":"${POSTGRES_PORT}"/postgres \
     -c "DROP DATABASE ${POSTGRES_DB};" -c "CREATE DATABASE ${POSTGRES_DB};" \
```

Comment:
removing this will break the NIH metrics dashboard right?

Reply:
yes, but only until PR 3/3 fixes them by using different metrics ;) Sorry about this, it goes together with changes in the NIH metrics dashboard yaml file in the other PR